source/testnet/net/resources/release-notes.rst

```diff
@@ -8,6 +8,103 @@ Release Notes
    :local:
    :backlinks: none
 
+
+.. _open-testnet-v5-update-2:
+
+Open Testnet v5 Update 2
+========================
+
+**Concordium Desktop Wallet v1.1.3 for Testnet.**
+
+The Desktop Wallet is available on Testnet for Windows, macOS, and Linux including:
+
+* All features released in v1.0.2 for Mainnet.
+* Transaction status is now available in account reports.
+* Various bug fixes.
+* Foundation feature: Added support for bulk import of proposals
+
+
+
+.. _open-testnet-v5-update-1:
+
+Open Testnet v5 Update 1
+========================
+
+June 24th, 2021
+
+:ref:`Concordium Mobile Wallet for iOS v1.0.5. <mainnet:downloads>`
+
+* Added feature enabling change of passcode and biometrics.
+* Updates to Account page UI for easier shielding/unshielding transactions.
+* Added option to filter rewards in transaction log.
+* Added About page.
+* Improved security.
+* Various bug fixes and robustness improvements.
+* Code is now open source.
+
+:ref:`Concordium Mobile Wallet for Android v1.0.7(46). <mainnet:downloads>`
+
+* Added feature enabling change of passcode and biometrics.
+* Updates to Account page UI for easier shielding/unshielding transactions.
+* Added option to filter rewards in transaction log.
+* Added About page.
+* Improved security.
+* Various bug fixes and robustness improvements.
+* Code is now open source.
+
+.. _open-testnet-v5:
+
+Open Testnet v5
+===============
+
+May 12th, 2021
+
+Updated Open Testnet to match Mainnet features, which includes:
+
+
+**Proof of Stake**
+
+The Concordium Blockchain uses a proof of stake mechanism to ensure resource-efficient operation of the network.
+
+
+**Two Layer Consensus Protocol**
+
+Nakamoto-Style Consensus Bakers participate in a form of lottery to win the right to append blocks to the chain.
```

Comment:
I think there's a missing colon (and it's also missing from the mainnet docs, so it should be changed there as well.)
Nakamoto-Style Consensus:  Bakers participate in